docs/assets/scss/_sidebar.scss

```
@@ -9,50 +9,60 @@
   top: calc(var(--navbar-height));
   margin-top: calc(var(--navbar-height));
   margin-right: calc(-1 * (var(--sidebar-width)));
-  
-  transform: translateX(calc(-1 * var(--sidebar-wrapper-width)));
-  transition: 0.5s ease-in-out, margin-right 0.5s ease-in-out;
+
+  transform: translateX(-100%);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)